espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/CreateSession.kt

```diff
@@ -34,9 +34,12 @@ class CreateSession : RequestHandler<SessionParams, Session> {
         try {
             val parsedCaps = parseCapabilities(params.capabilities.firstMatch,
                     params.capabilities.alwaysMatch)
-            val activityName = parsedCaps["appActivity"] as? String
-                    ?: throw InvalidArgumentException("appActivity capability is mandatory")
-            startActivity(parsedCaps["appPackage"] as? String, activityName)
+            val shouldLaunchApp = (parsedCaps["autoLaunch"] ?: true) as Boolean
+            if (shouldLaunchApp) {
```

Comment:
Was this change tested? As far as I remember the driver verifies that the given activity is running after the session has been started

Reply:
Yes, I give pass it the home package (launcher package) and '*' activity and it seems happy

Reply:
Perhaps it makes sense to skip activity startup verification if  `autoLaunch ` is set. The "real" package cap value is needed for other API calls

Reply:
Yes, added